docs/user-guide/ze-profiles.md

```diff
@@ -67,3 +86,27 @@ Activate the SCS plug-in in Zowe Explorer.
 6. Restart VSCode.
 
 The credentials of your newly created or existing profiles are now stored securely.
+
+## Use Base Profile and Token with Existing Profiles
+
+As a Zowe user, you can leverage the base profile functionality to access multiple Zowe services instantly. Base profiles enable you to authenticate via Zowe API Mediation Layer. You can use base profiles with more than one service profile. Any service profile with the same host and port as your base profile automatically uses your API ML token to log in.
```

Comment:
I don't know if we want to say "Zowe services". I think it can just be "services," as is described in this section: https://docs.zowe.org/stable/user-guide/cli-usingcli.html#using-profiles

Actually, if you can re-create much of the information from the above link, I think that would be helpful.

Reply:
With removing "Zowe" I think it reads fine. We can add more as we see confused questions on Slack ❓ ❔

Reply:
I guess by services I meant components/applications. If I replace "services" with any of those, will it be clearer?

Reply:
> Any service profile with the same host and port as your base profile automatically uses your API ML token to log in.

This is not true. I can put my host and port and username and password in my service profile and it will not use the token from my base profile.

Reply:
Some thoughts that might help (I worked on the CLI profile doc):
- For your first comment about services, I agree to omit the word "Zowe". Also keep in mind that you can only do SSO through APIML **if** the service has been configured/registered to the layer. For services that are not registered to the layer, you still need to supply host, port, etc... explicitly on commands or service profiles. 
- The benefit isn't that you can access services more "instantly". It's that you can point your commands at APIML to reap the benefits that API ML provides, such as SSO to multiple services, load balancing, security, etc... 
- I agree with @zdmullen comment that you might want to cherry-pick some of the CLI profiles doc and edit it as needed for Explorer doc. There are several important things described there that you might not want to write about from scratch, like base vs service profile, how the `--base-path` option works, how to do SSO but still use your service profiles to connect to regular services that aren't registered to APIML, etc.... Basically translate that doc to one about doing the same things using Explorer GUI.

Reply:
@jellypuno Agreed. It's more like "if you don't supply a host and port in your service profile, then the tool references your base profile and looks for a token to use instead. Omit the host and port from service profiles to "instruct it" to connect to APIML, along with providing --base-path of the APIML instance.

Reply:
@BrandonJenkins14 To your first point, wrt to accessing API ML, is this information (that the service has to be registered to layer to do SSO) available on the CLI/API ML page? I think I should link to it.
point 2 - right, thanks for clarifying this to me. I should change that part accordingly. 
last point - I guess I should do more research. This feature is more than meets the eye.

Reply:
@IgorCATech 

- https://docs.zowe.org/stable/extend/extend-apiml/onboard-overview.html  (an API developer or admin of some sort would likely do this) 
- check out the more recent comments, it seems that I went overboard with my suggestion to make significant changes. Might be close to complete here :P